nextcloudappstore/core/views.py

```diff
@@ -270,3 +273,42 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context['form'] = AppRegisterForm()
         return context
+
+
+class AppEditView(LoginRequiredMixin, TemplateView):
```

Comment:
Right, now I remember. I intended `AppEditView` to be able to support multiple forms (since the view is reached by clicking "Edit app"). Not sure if that is a good idea, but I suspect that views such as `UpdateView` only support one form in a meaningful way. Regardless, I will refactor this so that `post()` does not get stuff from the context.